pkgs/desktops/plasma-5.1/startkde/startkde.in

```diff
@@ -0,0 +1,361 @@
+#!@bash@/bin/bash -x
+#
+#  NIXOS KDE STARTUP SCRIPT
```

Comment:
@edolstra They still include that script, but it doesn't come close to working for us. I had to edit it so substantially that I thought it made more sense to include our own copy, rather than trying to maintain dozens of patches for a single script.